library/Core/src/Juvix/Core/IR/Evaluator/SubstV.hs

```diff
@@ -405,7 +416,9 @@ argToValue ::
   App.Arg (Param.PrimType primTy) primVal ->
   IR.Value primTy (Param.TypedPrim primTy primVal)
 argToValue = \case
-  App.TermArg (App.Take {type', term}) ->
-    IR.VPrim $ App.Return {retType = type', retTerm = term}
+  App.TermArg (App.Return {retType, retTerm}) ->
+    IR.VPrim $ App.Return {retType, retTerm}
+  App.TermArg (App.Cont {fun, args, numLeft}) ->
+    notImplemented
```

Comment:
We should probably note where all these unimplemented parts are and try to see if we can centralize this notion, as it seems to spread out all over the place, which makes maintaining harder.

Reply:
The `notImplemented` bits have to do with handling `Cont`. Could we leave the implementation of those to a separate PR?

Reply:
I'll make a PR myself after merging this, so I don't lose where this particular issue scatters to